app/react/components/InsightsBox/InsightsBox.tsx

```diff
@@ -0,0 +1,68 @@
+import clsx from 'clsx';
+import { Lightbulb, X } from 'lucide-react';
+import { ReactNode, useMemo } from 'react';
+import sanitize from 'sanitize-html';
+import { useStore } from 'zustand';
+
+import { Button } from '@@/buttons';
+
+import { insightStore } from './insights-store';
+
+export type Props = {
+  header: string;
+  content: ReactNode;
+  setHtmlContent?: boolean;
+  insightCloseId?: string; // set if you want to be able to close the box and not show it again
+};
+
+export function InsightsBox({
+  header,
+  content,
+  setHtmlContent,
+  insightCloseId,
+}: Props) {
+  // allow to close the box and not show it again in local storage with zustand
+  const { insightIDsClosed, addInsightIDClosed } = useStore(insightStore);
+  const isInsightClosed = useMemo(() => {
+    if (insightCloseId) {
+      return insightIDsClosed.includes(insightCloseId);
+    }
+    return false;
+  }, [insightCloseId, insightIDsClosed]);
```

Comment:
may I suggest adding a method on the store (`isClosed`), the receives an optional id:

```suggestion
  const insightStoreObj = useStore(insightStore);
  const isInsightClosed = insightStoreObj.isClosed(insightCloseId);
```